scripts/mkcloud

```diff
@@ -1179,8 +1176,10 @@ function sanity_checks()
         if the machine has a global public IP
         see bnc#928384 and CVE-2012-3411"
 
-    if [[ $want_sles12 ]] && [[ $want_sles12_controller ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 4 ]] ; then
-        complain 113 "Please increase number of nodes for this setup, minimal nodenumber=4"
+    if ! iscloudver 6plus; then
```

Comment:
If we want to go with full Controller node in SUSE Cloud 6, then we can remove whole this condition